src/rpc/blockchain.cpp

```diff
@@ -1031,23 +1008,22 @@ UniValue gettxout(const JSONRPCRequest& request)
                     {"include_mempool", RPCArg::Type::BOOL, /* default */ "true", "Whether to include the mempool. Note that an unspent output that is spent in the mempool won't appear."},
                 },
                 RPCResult{
-            "{\n"
-            "  \"bestblock\":  \"hash\",    (string) The hash of the block at the tip of the chain\n"
-            "  \"confirmations\" : n,       (numeric) The number of confirmations\n"
-            "  \"value\" : x.xxx,           (numeric) The transaction value in " + CURRENCY_UNIT + "\n"
-            "  \"scriptPubKey\" : {         (json object)\n"
-            "     \"asm\" : \"code\",       (string) \n"
-            "     \"hex\" : \"hex\",        (string) \n"
-            "     \"reqSigs\" : n,          (numeric) Number of required signatures\n"
-            "     \"type\" : \"pubkeyhash\", (string) The type, eg pubkeyhash\n"
-            "     \"addresses\" : [          (array of string) array of bitcoin addresses\n"
-            "        \"address\"     (string) bitcoin address\n"
-            "        ,...\n"
-            "     ]\n"
-            "  },\n"
-            "  \"coinbase\" : true|false   (boolean) Coinbase or not\n"
-            "}\n"
-                },
+                    RPCResult::Type::OBJ, "", "",
+                    {
+                        {RPCResult::Type::STR_HEX, "bestblock", "The hash of the block at the tip of the chain"},
+                        {RPCResult::Type::NUM, "confirmations", "The number of confirmations"},
+                        {RPCResult::Type::STR_AMOUNT, "value", "The transaction value in " + CURRENCY_UNIT},
+                        {RPCResult::Type::OBJ, "scriptPubKey", "",
+                            {
+                                {RPCResult::Type::STR_HEX, "asm", ""},
+                                {RPCResult::Type::STR_HEX, "hex", ""},
```

Comment:
Still learning how this community operates, should this MR also aim to update/fix descriptions?

Reply:
I want to make this change as easy to review as possible. So I am for a minimal rendered diff, assuming `--ignore-all-space`.

So for example, the diff for this RPC is:

```
$ git diff --ignore-all-space  --word-diff ./gettxout
diff --git a/gettxout b/gettxout
index 1d186cd..90fa947 100644
--- a/gettxout
+++ b/gettxout
@@ -8,21 +8,21 @@ Arguments:
3. include_mempool    (boolean, optional, default=true) Whether to include the mempool. Note that an unspent output that is spent in the mempool won't appear.

Result:
{                         {+(json object)+}
  "bestblock" : [-"hash",-]{+"hex",+}    (string) The hash of the block at the tip of the chain
  "confirmations" : n,    (numeric) The number of confirmations
  "value" : [-x.xxx,-]{+n,+}            (numeric) The transaction value in BTC
  "scriptPubKey" : {      (json object)
    "asm" : [-"code",-]{+"hex",+}        (string)
    "hex" : "hex",        (string)
    "reqSigs" : [-n,          (numeric)-]{+"hex",    (string)+} Number of required signatures
    "type" : [-"pubkeyhash",-]{+"hex",+}       (string) The type, eg pubkeyhash
    "addresses" : [       [-(array of string)-]{+(json array)+} array of bitcoin addresses
      [-"address"-]{+"str",+}              (string) bitcoin address
      [-,...-]
[-     ]-]{+...+}
{+    ],+}
  },
  "coinbase" : [-true|false   (boolean)-]{+"hex",     (string)+} Coinbase or not
}

Examples: